webui/react/src/components/DeterminedAuth.tsx

```diff
@@ -16,6 +17,9 @@ interface FromValues {
   username?: string;
 }
 
+const storage = new Storage({ basePath: '/DeterminedAuth', store: window.localStorage });
```

Comment:
question: How about using the `useStorage` hook? Will it not work because of the user namespace?

Reply:
yeah I think the login changes the namespace for that